src/components/layout/JobTemplate.tsx

```diff
@@ -0,0 +1,67 @@
+import Layout from "./Layout";
+import Link from "next/link";
+import Head from "next/head";
+import { useRouter } from "next/router";
+import siteConfig from "site.config";
+import TagList from "src/components/TagList";
+import { slugify } from "src/slugify";
+import PageMeta from "../PageMeta";
+import Tag from "../Tag";
+import { formatDistanceToNow } from "date-fns";
+import { dateFormat } from "src/dateFormat";
+
+type Props = {
+  frontMatter: any;
+  children?: any;
+};
+
+export default function PostTemplate({ frontMatter: post, children }: Props) {
+  const router = useRouter();
+  let editUrl = `${siteConfig.githubUrl}edit/master/src/pages/${router.pathname}/index.mdx`
+
+  return (
+    <Layout>
+      <PageMeta
+        title={post.title}
+        image={post.hero ? `${siteConfig.rootUrl}${post.hero}` : undefined}
+        description={post.excerpt}
+      />
+
+      <article className="article mt-8 lg:max-w-3xl mr-auto ml-auto">
+        <header className="inset mb-12">
+          <TagList tags={post.tags} />
+
+          <h1 className="hashtag mt-2 mb-1 text-4xl md:text-5xl font-bold leading-tight">
+            {post.title}
+          </h1>
+          {post.author && siteConfig.features.authorPages ? (
+            <span className="block text-base text-gray-600">
+              by{[" "]}
+              {post.author.map((author: string, i: number) => (
+                <span key={author}>
+                  <Link
+                    href="/author/[authorSlug]"
+                    as={`/author/${slugify(author)}`}
+                  >

```

Comment:
Now I think on it, further to the Tags issue - this may also cause us problems because I think the same logic will exist here as with Tags. In that AuthorPages will only list posts from the posts directory and/or won't resolve if they don't exist on a post somewhere :thinking:

Reply:
Data functions will need to support many to many relations and queries, I suppose. There are a few possibilities...

Reply:
GraphQL anyone?